source/_sass/infeedl.scss

```diff
@@ -93,12 +97,27 @@ h1, h2, h3, h4, h5, h6 {
     margin-left: 8.33333%;
     margin-right: 8.33333%;
   }
+  .sidebar-logo-wrap{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space